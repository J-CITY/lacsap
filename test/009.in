


function fac(n: integer): integer;
var
  p, i: integer;
begin
  p:=1;
  for i:=2 to n do
    p:=p*i;
    
  fac:=p;
end;

procedure express(a, b, c : real);
var
  z : real;
begin
  z:=a + b + c;
  x:=sqr(z);
  y:=sqrt(z);
end ;

program HelloWorld;
var
    int : integer;
    r : real;
begin
    int := 1;
end.
