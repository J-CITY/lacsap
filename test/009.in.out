                   ;   0   0       separator     5
